### PR TITLE
Add Multiple Dev Support

### PR DESCRIPTION
**Pull request by coopeeo**
# This Pull Request aims to add Multiple Developer support to the Geode SDK website mod list.

### CODE REVIEW

**Comment by RayDeeUx** [comment]
# we appreciate you using your outside voice to summarize the changes of this pull request

**Comment by coopeeo** [comment]
# THANK YOU VERY MUCH!

**Comment by Uproxide** [comment]
# why are we yelling!

**Comment by TechStudent10** [comment]
# because why not! (this needs to be added NOW thats why)

**Comment by matcool** [comment]
this code sux

File: build-mods-pages.js [hunk 1]
```diff
@@ -147,6 +147,20 @@ function filepath(parts) {
 }
 
 for (const mod of mods) {
+    if (mod.versions[0].modJSON.developer == undefined && mod.versions[0].modJSON.developers != undefined) {
```
Comment (matcool) [inline build-mods-pages.js:150]: jesus christ just make a variable for the current version

File: build-mods-pages.js [hunk 2]
```diff
@@ -147,6 +147,20 @@ function filepath(parts) {
...
+        if (mod.versions[0].modJSON.developers.length == 1) {
+            mod.versions[0].modJSON.developer = mod.versions[0].modJSON.developers[0]
+            mod.versions[0].modJSON.developertwo = mod.versions[0].modJSON.developers[0]
```
Comment (matcool) [inline build-mods-pages.js:153]: this is such a hack, just have it as a local variable

File: build-mods-pages.js [hunk 3]
```diff
@@ -147,6 +147,20 @@ function filepath(parts) {
...
+        } else if (mod.versions[0].modJSON.developers.length == 2) {
+            mod.versions[0].modJSON.developer = mod.versions[0].modJSON.developers[0] + " & " + mod.versions[0].modJSON.developers[1]
+            mod.versions[0].modJSON.developertwo = mod.versions[0].modJSON.developers[0] + " & " + mod.versions[0].modJSON.developers[1]
```
Comment (matcool) [inline build-mods-pages.js:156]: theres no reason for this to be different from the case with only 1 developer, both are just joining the array with &

File: build-mods-pages.js (hunk 1 above)
Comment (coopeeo) [reply build-mods-pages.js]: made some variables

File: build-mods-pages.js (hunk 3 above)
Comment (coopeeo) [reply build-mods-pages.js]: in a function

**Comment by coopeeo** [comment]
@matcool, I hope this is better!

**Comment by coopeeo** [comment]
^

**Comment by coopeeo** [comment]
me impatient!!!

**Comment by coopeeo** [comment]
May a Geode lead developer review my code to add multiple developer support.

**Pull request merged**